cli/src/ui.rs

```diff
@@ -382,15 +388,15 @@ impl Ui {
 
     /// Writer to print hint without the "Hint: " heading.
     pub fn hint_no_heading(&self) -> LabeledWriter<Box<dyn Formatter + '_>, &'static str> {
-        LabeledWriter::new(self.stderr_formatter(), "hint")
+        LabeledWriter::new(self.status(), "hint")
     }
 
     /// Writer to print hint with the given heading.
     pub fn hint_with_heading<H: fmt::Display>(
         &self,
         heading: H,
     ) -> HeadingLabeledWriter<Box<dyn Formatter + '_>, &'static str, H> {
-        HeadingLabeledWriter::new(self.stderr_formatter(), "hint", heading)
+        HeadingLabeledWriter::new(self.status(), "hint", heading)
```

Comment:
nit: I'm not sure whether the error hints should be suppressed, but I agree inline hints should be.

Reply:
Good catch. I've added a commit making `command_error.rs` not use the `hint*()` helpers.